commands/test/test-command.ts

```diff
@@ -44,6 +44,8 @@ const questions = [
     { question: 'Unternehmen sollen selbst entscheiden, ob sie ihren Beschäftigten das Arbeiten im Homeoffice erlauben.', tag: ['Arbeitsrecht', 'Digitalisierung'] },
 ];
 
+const specificQuestionMessage = {} as { [key: string]: any}
```

Comment:
this object is only stored in heap. therefore, it'd reset every time the bot re-deploys